src/kOS/Screen/TermWindow.cs

```diff
@@ -397,6 +452,24 @@ internal void AttachTo(SharedObjects shared)
         {
             this.shared = shared;
             this.shared.Window = this;
+            NotifyOfScreenResize(this.shared.Screen);
+            this.shared.Screen.AddResizeNotifier(NotifyOfScreenResize);
+        }
+        
+        internal int NotifyOfScreenResize(IScreenBuffer sb)
+        {
+            windowRect = new Rect(windowRect.xMin, windowRect.yMin, sb.ColumnCount*CHARSIZE + 65, sb.RowCount*CHARSIZE + 100);
+            return 0;
+        }
+        
+        private int HowManyRowsFit()
+        {
+            return (int)(windowRect.height - 100) / CHARSIZE;
+        }
+
+        private int HowManyColumnsFit()
+        {
+            return (int)(windowRect.width - 65) / CHARSIZE;
```

Comment:
These are meant to be the mapping from the window's pixel size to it's character cell size.